test/negative/memmove.with.phinode.src.ll

```diff
@@ -0,0 +1,59 @@
+; It is assumed that the test will fail
+
+; RUN: llvm-as %s -o %t.bc
+; RUN: not --crash llvm-spirv %t.bc -o %t.spv 2>&1 | FileCheck %s
+
+; CHECK: llvm.memmove of PHI instruction result not supported
+
+; ModuleID = 'test.bc'
+source_filename = "llvm-link"
+target datalayout = "e-i64:64-v16:16-v24:32-v32:32-v48:64-v96:128-v192:256-v256:256-v512:512-v1024:1024-n8:16:32:64"
+target triple = "spir64-unknown-unknown-sycldevice"
+
+%"group" = type { [1 x i64], [1 x i64], [1 x i64], [1 x i64] }
+%"accessor" = type { %"AccessorImplDevice", i32 addrspace(1)* }
+%"AccessorImplDevice" = type { [1 x i64], [1 x i64], [1 x i64] }
+%class.kfunc = type <{ %"accessor", %"group", i32, [4 x i8] }>
+
+$test_func_object = comdat any
+
+@InvocIndex = external local_unnamed_addr addrspace(1) constant i64, align 8
+@"func_object1" = internal addrspace(3) global %class.kfunc zeroinitializer, align 8
+
+; Function Attrs: convergent norecurse
+define weak_odr dso_local spir_kernel void @test_func_object() local_unnamed_addr comdat !kernel_arg_buffer_location !4 {
```

Comment:
I suppose that !kernel_arg_buffer_location metadata can be dropped if there are no arguments to a function.